packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -28,18 +43,52 @@
             oc-child-width-1-3@s
           "
         >
-          <a v-for="space in spaces" :key="space.id" href="#" class="oc-mb-m">
-            <span class="spaces-list-card oc-border oc-card oc-card-default">
-              <span class="oc-card-media-top oc-border-b">
+          <li v-for="space in spaces" :key="space.id" class="oc-mb-m">
+            <div class="spaces-list-card oc-border oc-card oc-card-default">
+              <div class="oc-card-media-top oc-border-b">
+                <oc-button
+                  :id="`space-context-btn-${space.id}`"
+                  v-oc-tooltip="$gettext('Show context menu')"
+                  :aria-label="$gettext('Show context menu')"
+                  class="oc-position-absolute oc-position-top-right oc-mr-s oc-mt-s"
+                >
+                  <oc-icon name="more-2" />
```

Comment:
Could be `size="small"` as it's quite prominent right now, design decision @tbsbdr

Reply:
those kind of buttons have a clickable size of 40x40 px in the design
nested icon size is 24x24
visual prominence should be taken back via less prominent bg color, but this can be part of design polishing imo
![image](https://user-images.githubusercontent.com/26610733/151536239-3fcfad77-6ce1-4ff1-9410-91ae937d2385.png)
![image](https://user-images.githubusercontent.com/26610733/151536337-00f2a4be-57b5-4479-8d10-2803035b1b24.png)